modules/nf-core/openms/idmerger/tests/main.nf.test

```diff
@@ -0,0 +1,61 @@
+nextflow_process {
+
+    name "Test Process OPENMS_IDMERGER"
+    script "../main.nf"
+    process "OPENMS_IDMERGER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "openms"
+    tag "openms/idmerger"
+
+    test("proteomics - openms - idxml") {
+
+        when {
+            process {
+                """
+                input[0] = Channel.of([
+                            [ id:'test'], // meta map
+                            [ file(params.test_data['proteomics']['openms']['idxml1'], checkIfExists: true),
+                              file(params.test_data['proteomics']['openms']['idxml2'], checkIfExists: true)]
+                        ])
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match("versions") },
+                { assert snapshot(process.out.idxml).match("idxml") }
```

Comment:
I think this is called filtered, not idxml

Reply:
the [process output declaration](https://github.com/nf-core/modules/blob/c65917d37cdaa9d6c26fccf8f7c313aab1a51d8a/modules/nf-core/openms/idmerger/main.nf#L14) emits `idxml`, so that should be accessible via `process.out.idxml` right?

also I did the unspeakable accidentally created a branch on the upstream instead of my fork. I aborted this PR and would reopen a new one from my modules fork in a minute.

Reply:
correct, was looking at the wrong module